src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.ComposeStore.targets

```diff
@@ -220,7 +220,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     <PropertyGroup>
       <DefaultComposeDir>$(HOME)</DefaultComposeDir>
       <DefaultComposeDir Condition="'$(OS)' == 'Windows_NT'">$(USERPROFILE)</DefaultComposeDir>
-      <DefaultComposeDir>$([System.IO.Path]::Combine($(DefaultComposeDir), '.dotnet', $(PlatformTarget), 'store'))</DefaultComposeDir>
+      <DefaultComposeDir>$([System.IO.Path]::Combine($(DefaultComposeDir), '.dotnet', 'store'))</DefaultComposeDir>
```

Comment:
Previously userprofile store was in
<user_profile>\<arch>\store

since we have removed both sdk and shared Fx from user_profile, this location no longer makes sense. I have moved store now under <user_profile>

The tooling adds <Arch> and <tfm> and passes this path as an additional probing path for developer scenarios